activities/code-jam/README.md

```diff
@@ -0,0 +1,58 @@
+
+# Goal
+
+We want to create an event that will both build camaraderie among existing members and allow us to reach out to new members and communities. This activity should be accessible to all skill levels by being light-hearted and fun rather than technically challenging. In addition, it should demonstrate that failure is OK and reward participants for trying new things, one of the core principles of RCOS.
+
+# Overview
+
+The RCOS Code Jam will be a semi-competitive fun event where students compete in teams or by themselves on a small programming project. At the end of the event, students will be awarded points based on some criteria. Some of the criteria would be based on the project itself, though most of the criteria will be subjectively based on creativity or other fun aspects. The event itself is meant to be an inclusive meeting to foster internal relationships between current and potential RCOS students. 
+
+# Event Structure
+
+The event will take place over a 6-8 hour period, similar to a single-day hackathon. At the beginning of the event, participants are provided with a prompt. This prompt will be chosen from a list of potential prompts by popular vote of the participants. This prompt should be sufficiently vague to allow much room for interpretation, but focused enough to allow us to compare solutions. The prompt should ask participants to solve a problem, but the interpretation of the problem itself and how it should be solved is up to the participants. This will (hopefully) allow participants to apply whatever skills they possess without feeling overwhelmed or unqualified. Solutions may take the form of a web service, a video game, a visualization, an interactive command line tool, or anything the participants can think of. Participants may work individually or in small groups. Lunch, snacks, and beverages will be provided for all participants.
+
+# Potential Projects
+
+Participants will vote at the start of the event on which prompt they would like to solve, but all participants should follow the same prompt. Some potential prompts include (details to come):
+
+- Build a sandwich factory
+- Emoji Recomremender 😄 
+- Manage a farm
+- Plan a vacation
+- Build a railroad
+- An absurd campus-lore related challenge
+- Place like 10 more ideas here
+# Mentor Duties
+
+Mentors should be present and floating around the event. Mentors should help participants form teams, decide what they want to do, and provide inspiration tailored to the specific skills of each participant as needed. Mentors should introduce participants to relevant technologies they might find useful and accessible based on their background and what they want to accomplish. Some ideas for potential solutions should be pre-prepared and suggested to struggling groups as needed.
```

Comment:
Should there be an optional workshop (or workshops) on some technology (or technologies) we think might be useful? It would be good for new/prospective members without any experience, and could make the event more welcoming to less confident students. It would also probably help decrease the workload for mentors.

If you remember what Wes posted a while back, the "coding workshop" brought in a more diverse selection of participants. Maybe we could try implementing that here?